instrumentation/grape/example/trace_demonstration.rb

```diff
@@ -43,8 +43,11 @@ class ExampleAPI < Grape::API
 # Set up fake Rack application
 builder = Rack::Builder.app do
   # Integration is automatic in web frameworks but plain Rack applications require this line.
+  # - middleware_args_old to emit old HTTP semantic conventions
+  # - middleware_args_stable to emit stable HTTP semantic conventions
+  # - middleware_args_dup to emit both old and stable HTTP semantic conventions
   # Enable it in your config.ru.
-  use *OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args
+  use *OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args_old
```

Comment:
Updating this Grape example to patch the old module.

Reply:
I don't think so. The original example looks like this is a config line that users must add either way!